src/catchup/VerifyLedgerChainWork.cpp

```diff
@@ -94,46 +95,48 @@ VerifyLedgerChainWork::getStatus() const
     {
         std::string task = "verifying checkpoint";
         return fmtProgress(mApp, task, mRange.first(), mRange.last(),
-                           mCurrCheckpoint);
+                           (mRange.last() - mCurrCheckpoint));
     }
     return Work::getStatus();
 }
 
 void
 VerifyLedgerChainWork::onReset()
 {
-    auto setLedger = mApp.getLedgerManager().getLastClosedLedgerHeader();
-    if (setLedger.header.ledgerSeq < 2)
-    {
-        setLedger = {};
-    }
-    if (mFirstVerified.header.ledgerSeq != 0)
-    {
-        mFirstVerified = setLedger;
-    }
-    if (mLastVerified.header.ledgerSeq != 0)
+    // Note that mFirstVerified is not reset here, since it is set
+    // in the end of verification, where work wouldn't fail
+
+    if (mVerifiedAhead.header.ledgerSeq != 0)
```

Comment:
why the condition on resetting `mVerifiedAhead` ?

Reply:
This was in the original code, looks like it's unnecessary, so I removed it.